packages/file-ops/main/index.js

```diff
@@ -53,6 +53,14 @@ class FileOpsChannel extends IpcChannel {
       this.exec(`gnome-terminal --working-directory=${filePath}`)
     }
   }
+
+  async trashFile (filePath) {
+    try {
+      await shell.trashItem(path.normalize(filePath))
```

Comment:
the `trash` command delete folder failed if the folder level is too deep in the Windows OS, so use `shell.trashItem` to move a file to recycle bin instead of trash package
ref: https://github.com/microsoft/vscode/blob/ad5ef7ea85b8a622f73b96394507d98809e68861/src/vs/platform/files/electron-main/diskFileSystemProviderServer.ts#L48